core/state/database.go

```diff
@@ -370,7 +369,7 @@ func (db *cachingDB) OpenTrie(root common.Hash) (Trie, error) {
 
 		// Otherwise, return a transition trie, with a base MPT
 		// trie and an overlay, verkle trie.
-		mpt, err = db.openMPTTrie(db.baseRoot)
+		mpt, err := db.openMPTTrie(db.baseRoot)
```

Comment:
:thinking:

Reply:
doesn't matter, but in the spirit of keeping the diff smaller, I'll revert.
```suggestion
		mpt, err = db.openMPTTrie(db.baseRoot)
```